### PR TITLE
clarified that buildx is not included in all linux distros

### PR DESCRIPTION
**Pull request by admackin**
As noted in #132 there is a lot of confusion from people expecting it to be present. I see [there is code to install it in docker CE](https://github.com/docker/docker-ce/blob/b63a7966fe7bc67dc15403c29f41f060ba8cb44a/components/packaging/plugins/buildx.installer) but it seems that linux distros (at least Amazon Linux for me) don't always include this even with version 19.03. I think this update reflects the true state of things.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Using `buildx` as a docker CLI plugin requires using Docker 19.03. A limited set
 
 ### Docker CE
 
-`buildx` comes bundled with Docker CE starting with 19.03, but requires experimental mode to be enabled on the Docker CLI.
+`buildx` comes bundled by default with Docker Desktop Docker CE starting with 19.03, but may not be present in all repackaged versions for Linux distributions (in which case follow the 'Binary release' instructions below). When it is included, it requires experimental mode to be enabled on the Docker CLI.
```
Comment (AkihiroSuda) [inline README.md:48]: "Docker Desktop Docker CE" -> "Docker Desktop"

Comment (admackin) [inline README.md:48]:
```suggestion
`buildx` comes bundled by default with Docker Desktop and Docker CE starting with 19.03, but may not be present in all repackaged versions for Linux distributions (in which case follow the 'Binary release' instructions below). When it is included, it requires experimental mode to be enabled on the Docker CLI.
```

**Comment by prutschman** [comment]
Merging this would have saved me a couple hours :|

**Comment by crazy-max** [comment]
@admackin Needs DCO and rebase

File: README.md [hunk 2]
```diff
@@ -49,7 +49,9 @@ Using `buildx` as a docker CLI plugin requires using Docker 19.03 or newer. A li
 
 ### Docker
 
-`buildx` comes bundled with Docker Desktop and in latest Docker CE packages.
+`buildx` comes bundled by default with Docker Desktop, and with Docker CE packages starting with 19.03, but may not be present in all repackaged versions for Linux distributions (in which case follow the 'Binary release' instructions below).
```
Comment (admackin) [inline README.md:52]:
```suggestion
`buildx` comes bundled by default with Docker Desktop (and with Docker CE packages) starting with 19.03, but may not be present in all repackaged versions for Linux distributions (in which case follow the 'Binary release' instructions below). 
```

**Pull request closed without merging**